script.skin.info.service/Utils.py

```diff
@@ -301,17 +302,37 @@ def clear_properties():
         HOME.clearProperty('SkinInfo.Album.Songs.Count')
 
 
-def passDataToSkin(name, data, prefix="", debug=False):
-    if data is not None:
-       # log( "%s%s.Count = %s" % (prefix, name, str(len(data)) ) )
-        for (count, result) in enumerate(data):
-            if debug:
-                log("%s%s.%i = %s" % (prefix, name, count + 1, str(result)))
-            for (key, value) in result.iteritems():
-                HOME.setProperty('SkinInfo.%s%s.%i.%s' % (prefix, name, count + 1, str(key)), unicode(value))
-                if debug:
-                    log('%s%s.%i.%s --> ' % (prefix, name, count + 1, str(key)) + unicode(value))
-        HOME.setProperty('SkinInfo.%s%s.Count' % (prefix, name), str(len(data)))
-    else:
-        HOME.setProperty('SkinInfo.%s%s.Count' % (prefix, name), '0')
-        log("%s%s.Count = None" % (prefix, name))
+# def Notify(header, line='', line2='', line3=''):
+#     xbmcgui.Dialog().notification(('%s, %s, %s, %s') % (header, line, line2, line3))
+
+
+# def GetStringFromUrl(encurl):
+#     succeed = 0
+#     while succeed < 5:
+#         try:
+#             req = urllib2.Request(encurl)
+#             req.add_header('User-agent', 'Mozilla/5.0 (Windows; U; Windows NT 5.1; en-GB; rv:1.9.0.3) Gecko/2008092417 Firefox/3.0.3')
+#             res = urllib2.urlopen(req)
+#             html = res.read()
+#             return html
+#         except:
+#             log("GetStringFromURL: could not get data from %s" % encurl)
+#             xbmc.sleep(1000)
+#             succeed += 1
+#     return ""
+
+
+# def passDataToSkin(name, data, prefix="", debug=False):
+#     if data is not None:
+#        # log( "%s%s.Count = %s" % (prefix, name, str(len(data)) ) )
+#         for (count, result) in enumerate(data):
+#             if debug:
+#                 log("%s%s.%i = %s" % (prefix, name, count + 1, str(result)))
+#             for (key, value) in result.iteritems():
+#                 HOME.setProperty('SkinInfo.%s%s.%i.%s' % (prefix, name, count + 1, str(key)), unicode(value))
+#                 if debug:
+#                     log('%s%s.%i.%s --> ' % (prefix, name, count + 1, str(key)) + unicode(value))
+#         HOME.setProperty('SkinInfo.%s%s.Count' % (prefix, name), str(len(data)))
+#     else:
+#         HOME.setProperty('SkinInfo.%s%s.Count' % (prefix, name), '0')
+#         log("%s%s.Count = None" % (prefix, name))
```

Comment:
It's not my project, but doesn't your version control system provide you all previous versions of the code?